frontend/app/locales/de.json

```diff
@@ -53,7 +53,7 @@
   "comment.verified-user": "Bestätigter Benutzer",
   "comment.verify-user": "Möchtest du den Benutzer {userName} wirklich bestätigen?",
   "comment.vote-error": "Fehler beim Abstimmen: {voteErrorMessage}",
-  "commentForm.anonymous-uploading-disabled": "Der Upload von Bildern ist nur für registrierte Benutzer möglich. Bitte melde dich zuvor an.",
+  "commentForm.unauthorized-uploading-disabled": "Der Upload von Bildern ist nur für registrierte Benutzer möglich. Bitte melde dich zuvor an.",
```

Comment:
Lang generation is broken, It's dropped message for `anonymous-uploading-disabled`.
I think it happened because the generator is not able to extract translation when you get it with trinary operator.